### DISCUSSION TITLE
--param use-hw-accel no longer valid

### DISCUSSION THREAD

**Discussion by alatteri** [CESNET/UltraGrid#234] [General]
Hello,

This parameter is no longer valid.  Utilizing this previously on the decoder helped a lot to reduce the load on the machine.  Is there a new approach to this that no longer requires the explicit param?

Thanks,
Alan

**Comment by alatteri** [comment]
I don't remember what this is all about, but I'm not getting the error anymore, so ignore this I guess.

**Reply by MartinPulec** [reply]
Oops, I've perhaps missed this discussion. Thanks for the info.

Even if I turn back to it, I also don't know - this hasn't been changed recently. It's true, that the parameter itself is conditionally enabled. But the condition is `libavcodec libavutil >= 55.22.1`, so it should be actually present in any FFmpeg-enabled builds.
